tracks/javascript/exercises/resistor-colors/mentoring.md

```diff
@@ -0,0 +1,132 @@
+This exercise introduces `map` and `join` to the student, as well as converting from and to `number`, and builds upon
+their knowledge of arrays from the previous exercise `resistor-color`.
+
+### Reasonable solutions
+
+```javascript
+export const COLORS = [
+  'black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet',
+  'grey', 'white',
+];
+
+export function value(colors) {
+  return Number(colors.map(color => COLORS.indexOf(color)).join(''))
+}
+```
+
+A student may choose to enter the `function` into `map`:
+
+```javascript
+colors.map(COLORS.indexOf.bind(COLORS))
+```
+
+A student may use also use an arrow function expression instead of a function declaration, as well as using an
```

Comment:
This still needs fixing - `may also use an arrow function`

Reply:
I don't know what happened there